src/main/java/tutorease/address/logic/commands/ListContactCommand.java

```diff
@@ -17,10 +17,10 @@ public class ListContactCommand extends ContactCommand {
     @Override
     public CommandResult execute(Model model) {
         requireNonNull(model);
+        model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
```

Comment:
This is essential for users to go back to the entire list after they search a keyword that does not have any match!